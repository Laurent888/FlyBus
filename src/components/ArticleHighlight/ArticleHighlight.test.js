import React from "react";
import { shallow } from "enzyme";
import ArticleHightlight from "./ArticleHighlight";
import image from "../../img/a320-thumb.jpg";

describe("Article hightlight", () => {
  const image2 = [image];
  const title = "Test Title";
  const url = "http://test.com";

  it("should render article hightlight", () => {
    const wrapper = shallow(
      <ArticleHightlight image={image2} title={title} url={url} />
    );

    expect(wrapper.length).toBe(1);
    expect(wrapper).toMatchSnapshot();
  });
});
