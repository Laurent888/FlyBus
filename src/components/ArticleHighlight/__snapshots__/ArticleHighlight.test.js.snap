// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Article hightlight should render article hightlight 1`] = `
<div
  className="article-highlight row"
>
  <div />
  <img
    alt="flybus-img"
    src="a320-thumb.jpg"
    style={
      Object {
        "height": "100%",
        "objectFit": "cover",
        "objectPosition": "center",
        "width": "100%",
      }
    }
  />
  <div
    className="article-highlight__content"
  >
    <h3
      className="lg-title"
    >
      Test Title
    </h3>
    <ButtonTransparent
      className="btn"
      label="Learn More"
      url="http://test.com"
    />
  </div>
</div>
`;
